ajax/libs/roundabout/package.json

```diff
@@ -19,5 +19,6 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/fredleblanc/roundabout.git"
-  }
+  },
+  "license": "BSD-3-Clause",
```

Comment:
comma "," have to be removed. It will be
```
"license": "BSD-3-Clause"
```
We don't need comma when there is no other fields.
For example, after "repository", there is still "license", so we need comma to separate them. But there is no field after "license", therefore comma is not needed.

You don't have to open an new pull request when you do the modification, just do it on the same branch of this pull request. In you case, it's `update-roundabout` branch. Update it and commit again. For your reference, commit message can be 
```
Remove spare comma to add license correctly, cc #11931
```
![s](https://user-images.githubusercontent.com/13430892/31166562-c9c522ea-a921-11e7-81c7-779b41cf6c76.png)